tools/wurf_options.py

```diff
@@ -63,6 +66,20 @@ def resolve(ctx):
 
 class WurfOptions(Options.OptionsContext):
 
+    def execute(self):
+
+        # Execute the context, this will go out and invoke the option(...)
+        # function defined in the wscript. We have some extra tools that
+        # also define the option(...) function to avoid having to call load
+        # in all wscripts to initialize those, we can do it here.
+
+        self.load('wurf_common_tools')
```

Comment:
You are right, maybe we can also control the loaded tools with some environment variables (if we get a situation where a particular tool is incompatible with a project). But let's cross that bridge when we get there.

I'm not sure if hiding the load operation is a good idea in any step, but you can experiment with that solution before this change is merged :)
Currently, we have a great explicit solution and it is easy to follow what is loaded where. We don't gain much by removing those 4 lines, and we definitely make things more obscure!